### ISSUE TITLE
Add a parameter to suppress show sketch_local

### ISSUE THREAD

**Issue by bernhard-42** [bernhard-42/vscode-ocp-cad-viewer#59]
Jern's approach:

```python
classes = (BuildPart, BuildSketch, BuildLine)  # for OCP-vscode
variables, s_o, s_n, slocal = (list(cf().f_locals.items()), [], [], False)
for name, obj in variables:
    if (
        isinstance(obj, classes)
        and not name.startswith("_")
        and not name.startswith("obj")
    ):
        if obj._obj_name != "sketch" or slocal:
            s_o.append(obj), s_n.append(f"{name}.{obj._obj_name}")
        elif obj._obj_name == "sketch":
            s_o.append(obj.sketch), s_n.append(f"{name}.{obj._obj_name}")
show(
    *s_o,
    # ,
    names=s_n,
    reset_camera=Camera.KEEP,
)
```

**Comment by jdegenstein** [comment]
missing import statement for the above:
```py
from inspect import currentframe as cf
```

**Comment by bernhard-42** [comment]
fixed in 2.2.2
